docs/content/tools/plotCorrelation.rst

```diff
@@ -1,8 +1,33 @@
 plotCorrelation
-===============
+==========================
 
 .. argparse::
-   :ref: deeptools.plotCorrelation.parse_arguments
-   :prog: plotCorrelation
+   :ref: deeptools.bamFingerprint.parse_arguments
+   :prog: bamFingerprint
```

Comment:
@bgruening sorry forgot to edit the argparse line.. It's plotCorrelation.. Fixed now.